codespell_lib/data/dictionary.txt

```diff
@@ -1992,6 +1992,8 @@ adminitrative->administrative
 adminitratively->administratively
 adminitrator->administrator
 adminitrators->administrators
+adminsitrator->administrator
+adminsitrators->administrators
```

Comment:
The surrounding suggestions included `administer(ed/ing/s)`, `administration(s)`, and `administrative(ly)`. Please add these words.

Reply:
Fixed.

Reply:
@int-y1 thank you for catching that, and @DimitriPapadopoulos thank you for fixing those!

Reply:
You're welcome, thank you for all the improvements so far.